src/tray.js

```diff
@@ -61,14 +63,38 @@ exports.create = function(config) {
         },
     ]);
 
+    blinkEnabled = config.blinkEnabled;
+
     const defaultIcon = nativeImage.createFromPath(config.icon_path);
 
     trayIcon = new Tray(defaultIcon);
     trayIcon.setToolTip(config.brand);
     trayIcon.setContextMenu(contextMenu);
     trayIcon.on('click', toggleWin);
 
+
     let lastFavicon = null;
+
+    let blinkInterval = null;
+    const currentIcon = {icon: defaultIcon};
+    const emptyIcon = nativeImage.createFromBitmap(Buffer.alloc(4, 0), {width: 1, height: 1});
+    ipcMain.on('startBlinking', function() {
+        if (!blinkEnabled || blinkInterval) {
+            return;
+        }
+        let iconState = false;
+        blinkInterval = setInterval(() => {
+            trayIcon && trayIcon.setImage(iconState ? currentIcon.icon : emptyIcon);
+            iconState = !iconState;
+        }, 500);
+    });
+    ipcMain.on('stopBlinking', function() {
+        if (blinkInterval) {
+            clearInterval(blinkInterval);
+            trayIcon && trayIcon.setImage(currentIcon.icon);
```

Comment:
Unsure whether it's codified in our code style but I don't see any reason to prefer this syntax over a single-line `if` statement.

Reply:
I changed the affected lines to `if (trayIcon) ...`